main.go

```diff
@@ -6,44 +6,43 @@ import (
 	"os"
 	"strconv"
 
+	log "github.com/sirupsen/logrus"
 	v1 "k8s.io/api/batch/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
-
-	log "github.com/sirupsen/logrus"
 )
 
-func FindExpiredJobs(jobList []v1.Job, annotation string, validatorList []JobValidator) []v1.Job {
+func FindExpiredJobs(jobs []v1.Job, annotation string, validatorList []JobValidator) []v1.Job {
 	expiredJobs := []v1.Job{}
 
 	ignoreAnnotationName := fmt.Sprintf("%s/ignore", annotation)
 
-	log.Debugf("(%d) jobs found", len(jobList))
+	log.Debugf("(%d) jobs found", len(jobs))
 
-	for _, job := range jobList {
-		ignoreAnnotation := job.ObjectMeta.Annotations[ignoreAnnotationName]
+	for i := range jobs {
```

Comment:
Is there clear guidance on iteration in go? Personally I'm a fan of unpacking each value into a single variable rather than indexing with `i`. That's just the "pythonista" in me though.